tests/executables_src/testUserInputValidator.cc

```diff
@@ -0,0 +1,640 @@
+// SPDX-FileCopyrightText: Deutsches Elektronen-Synchrotron DESY, MSK, ChimeraTK Project <chimeratk-support@desy.de>
+// SPDX-License-Identifier: LGPL-3.0-or-later
+#include <future>
+
+#define BOOST_TEST_MODULE testUserInputValidator
+
+#include "Application.h"
+#include "ScalarAccessor.h"
+#include "TestFacility.h"
+#include "UserInputValidator.h"
+
+#include <boost/mpl/list.hpp>
+#include <boost/test/included/unit_test.hpp>
+
+using namespace boost::unit_test_framework;
+namespace ctk = ChimeraTK;
+
+/*********************************************************************************************************************/
+/* Test module with a single validated input, used stand alone or as a downstream module *****************************/
+/*********************************************************************************************************************/
+
+struct ModuleA : public ctk::ApplicationModule {
+  using ctk::ApplicationModule::ApplicationModule;
+
+  ctk::ScalarPushInputWB<int> in1{this, "in1", "", "First validated input"};
+
+  ctk::UserInputValidator validator;
+
+  static constexpr const char* in1ErrorMessage = "in1 needs to be smaller than 10";
```

Comment:
Wonder if we should start using string_view for our constexpr strings. It's safe to use .data() on that because the string is static, or just create a std::string on-the-fly

Reply:
somehow that did not work in this context and I gave up... Maybe I forgot the `.data()`, will check.

Reply:
works with `.data()` and hence corrected...